source/omni_order_api.rst

```diff
@@ -0,0 +1,1249 @@
+************************************
+Omni Order API
+************************************
+
+
+`1. Sync Order`_
+
+`2. Get Order`_
+
+`3. Get Order Details`_
+
+`4. Confirm Order`_
+
+`5. Mark Order as Paid`_
+
+`6. Complete Order`_
+
+`7. Cancel Order`_
+
+
+1. Sync Order
+====================
+| URL endpoint: https://host.com/api/store/orders/sync
+| Allowed method: GET
+| Requires authentication: Yes
+
+A. Request
+----------
+
+You get all new orders which have not been synced before by calling the API with these parameters
+
+It will only return said orders only **once**.
+
+The store is derived from the store token string.
+
+Header
+______
+
+=================== =========== =======================
+Parameter           Required    Description
+=================== =========== =======================
+Content-Type        Yes         application/json
+Authorization       Yes         store token string
+=================== =========== =======================
+
+.. code-block::
+
+    Content-Type: application/json
+    Authorization: token vE53k50FVtct50ll8iHBE6FgMRVCyJeF
+
+Example of API call request using cURL (JSON). To avoid HTTP 100 Continue, please specify "Expect:" as a header.
+
+.. code-block:: bash
+
+    $ curl -X GET -H "Content-Type: application/json" -H "Authorization: token vE53k50FVtct50ll8iHBE6FgMRVCyJeF" -H "Expect:" https://host.com/api/store/orders/sync
+    
+
+B. Response
+-----------
+
+In response to these API calls, Omni will reply with the following data in JSON:
+
+=================== ==================
+Variable            Description
+=================== ==================
+orders              An array of order objects
+=================== ==================
+
+**ALL** new orders will be synced, but Omni only replies with an array of up to 100 order objects which have not been synced, and each order object has the following `fields`_.
+
+The order objects will only be returned **once**.
+
+If all orders are synced, it returns an empty array :code:`{ "orders" : [] }`
```

Comment:
`If there are no new orders, it returns an empty array :code:`{ "orders" : [] }``